backend/app/models/score.rb

```diff
@@ -1,3 +1,7 @@
 class Score < ApplicationRecord
   belongs_to :user
+
+  def self.revenue_with_username
```

Comment:
Is this for a future refactor? I think that this method shouldn't be here unless used.